common/internal_model/src/main/scala/weco/catalogue/internal_model/index/WorksIndexConfig.scala

```diff
@@ -76,6 +76,9 @@ object WorksIndexConfig extends IndexConfigFields {
       val identifiersPath = List("search.identifiers")
       val titlesAndContributorsPath = List("search.titlesAndContributors")
 
+      val newIdentifiersPath = List("query.allIdentifiers")
+      val newTitlesAndContributorsPath = List("query.titlesAndContributors")
```

Comment:
Assuming these distinctions are temporary?

Reply:
Yeah, I’ll drop the `new` suffix when I remove the old fields, but I didn't want to delete the old `search.*` fields quite yet.